src/appforms/tests/modules.js

```diff
@@ -43,7 +44,35 @@
   "060-RuleEngine"
  ],
  backbone:[
-  "040-view06Steps"
```

Comment:
That is true, but to be able to test some fields, it has to load the view file also.

If the test file doesn't exist, it won't fail any tests.